### PR TITLE
add application version number information

### PR DESCRIPTION
**Pull request by UrosPurtic**

### CODE REVIEW

File: src/Data/LoggerAbstract.php [hunk 1]
```diff
@@ -103,6 +107,17 @@ public function setExcluded(Exclude $exclude)
         $this->exclude = $exclude;
     }
 
+    public function setAppVersionNumber($version)
```
Comment (ppavlovic) [inline src/Data/LoggerAbstract.php:110]: Add type hint:
`public function setAppVersionNumber(Version $version)`

File: src/Data/LoggerAbstract.php [hunk 2]
```diff
@@ -103,6 +107,17 @@ public function setExcluded(Exclude $exclude)
...
+    {
+        $this->version = $version;
+        return $this;
+    }
+
+    public function getAppVersionNumber()
+    {
+        return $this->version instanceof Version ? $this->version->getVersionNumber() : $this->version;
```
Comment (ppavlovic) [inline src/Data/LoggerAbstract.php:118]: Did you mean to write something like:
`return $this->version instanceof Version ? $this->version->getVersionNumber() : null`

File: src/Data/TaskerEnd.php [hunk 3]
```diff
@@ -21,6 +21,7 @@ public function getRawData()
             'exec_time'    => $this->getElapsedTime(),
             'exec_time_ms' => (int) ($this->getElapsedTime() * 1000),
             'php_version'  => str_replace(PHP_EXTRA_VERSION, '', PHP_VERSION),
+            'app_version'        => $this->getAppVersionNumber(),
```
Comment (ppavlovic) [inline src/Data/TaskerEnd.php:24]: when you are so careful about indentation, fix this :)

File: src/Data/LoggerAbstract.php [hunk 4]
```diff
@@ -103,6 +107,17 @@ public function setExcluded(Exclude $exclude)
         $this->exclude = $exclude;
     }
 
+    public function setAppVersionNumber($version)
+    {
+        $this->version = $version;
+        return $this;
+    }
+
+    public function getAppVersionNumber()
```
Comment (ppavlovic) [inline src/Data/LoggerAbstract.php:116]: Add type hint
```
/**
 * @return string|null
*/
```

**Review by ppavlovic — CHANGES_REQUESTED** [review]

File: src/Data/LoggerAbstract.php (hunk 2 above)
Comment (UrosPurtic) [reply src/Data/LoggerAbstract.php:118]: No. $this->version can be null by itself so no need to do ": null". I'm just making the check to make sure the ->getVersionNumber method doesn't break

**Pull request merged**